docs/source/user-manual/copy-paste-blocks.md

```diff
@@ -0,0 +1,43 @@
+---
+myst:
+  html_meta:
+    "description": "Copy, cut and paste blocks in Volto, the Plone 6 frontend."
+    "property=og:description": "Copy, cut and paste blocks in Volto, the Plone 6 frontend."
+    "property=og:title": "How to copy, cut, and paste Volto blocks"
+    "keywords": "Volto, Plone, frontend, React, User manual, edit blocks, copy, cut, paste"
+---
+
+(copy-cut-paste-blocks-label)=
+
+# Copy, Cut, and Paste blocks in Volto
+
+Volto supports copying/cutting of blocks from one page and pasting them to another page. Here's how you can do:
+
+(copy-cut-blocks-label)=
+
+## Copy/Cut blocks
+
+While in edit mode you can hold <kbd>ctrl</kbd> key to select multiple blocks you want to copy/cut.
+Once blocks are selected, you will have a copy <i class="fas fa-copy"></i> and a cut <i class="fas fa-cut"></i> option at the bottom of Toolbar.
```

Comment:
We should use the [Volto (Pastanaga) icons](https://pastanaga.io/icons/), not Font Awesome icons. We want any icons used inline to be consistent with the actual UI.

This also requires a new MyST extension. See https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#html-images

Here is untested sample markup. Also I know I just said that HTML should be avoided, but that MyST extension supposedly resolves the issue of different output formats.

```suggestion
Once blocks are selected, you will have a copy <img alt="../_static/copy.svg" src="../_static/copy.svg" style="height: 1em;"> and a cut <img alt="../_static/cut.svg" src="../_static/cut.svg" style="height: 1em;"> option at the bottom of Toolbar.
```

Reply:
Sure will do!